src/components/ZetkinPerson.tsx

```diff
@@ -14,7 +14,11 @@ const ZetkinPerson: React.FunctionComponent<{
             <Avatar src={ orgId ? `/api/orgs/${orgId}/people/${id}/avatar` : '' } />
             <Box alignItems="start" display="flex" flexDirection="column" justifyContent="center" ml={ 1 }>
                 <Typography variant="body1">{ name }</Typography>
-                { typeof subtitle === 'string' ? <Typography>{ subtitle }</Typography> : <Box alignItems="center" display="flex">{ subtitle }</Box> }
+                { typeof subtitle === 'string' ?
+                    <Typography>{ subtitle }</Typography>
+                    :
+                    <>{ subtitle }</>
```

Comment:
The fragment should not be needed here. I'm guessing you put it here so that you could use `{ subtitle }` (because the `{}` is only meaningful in JSX) but another way would have been to just remove `{}` and put just `subtitle` here.